controller/script/destination-client/main.go

```diff
@@ -68,6 +68,9 @@ func get(client pb.DestinationClient, req *pb.GetDestination) {
 				default:
 					log.Printf("  - protocol hint: UNKNOWN")
 				}
+				if ot := addr.GetProtocolHint().GetOpaqueTransport(); ot != nil {
+					log.Printf("  - opaque transport port: %d", ot.GetInboundPort())
+				}
```

Comment:
This change is related to running the destination service locally and checking the `opaque_transport` field for destination profile updates.